### PR TITLE
Use yarn in CI

### PR DESCRIPTION
**Pull request by watrool**
Connected to https://github.com/rangle/rangle-starter/issues/189

### CODE REVIEW

**Comment by watrool** [comment]
@SethDavenport Just noticed that CI has not been enabled here ...

**Comment by winkerVSbecks** [comment]
@watrool I enabled CI for this. Can you close and resubmit the PR. Thanks

File: circle.yml [hunk 1]
```diff
@@ -10,12 +10,18 @@ dependencies:
...
     - pip install robotframework-requests==0.4.4
     - pip install robotframework-selenium2library==1.7.4
-    # Clean out node-modules to prevent issues with circle-ci's caching.
-    - rm -rf node_modules
+    - npm install -g yarn@0.16.1
+  override:
+    - yarn install
 test:
+  pre:
+    - yarn run clean
```
Comment (SethDavenport) [inline circle.yml]: you don't need the `run` with `yarn` :)

**Comment by winkerVSbecks** [comment]
squash and :shipit:

**Pull request merged**